Pull request from screen it report branch

elisabascunan opened the pull request:
This branch is ahead of main by one commit that includes:

1) the updated version of the descriptive-analysis.qmd file, with updated tables and 2 new analyses (comparation table of all the tools plus location analyses of false positives.

2) addition of the two datasets (regset.csv and nc_man_check.csv) to the folder data.

Commit:
This commit includes:
1) the updated version of the descriptive-analysis.qmd file, with updated tables and new analyses (comparation of tools plus location analyses of false positives.

2) addition of the two datasets (regset.csv and nc_man_check.csv).

delwen commented on descriptive-analysis.qmd:
````diff
@@ -175,41 +209,46 @@ The IDs analyzed could be in **one or more locations**, that is why there are mo
+  kable()
 ```
 
 ## 5) Description of each tool - type of ID and location
````
@elisabascunan overall comment for this section: when you refer to the location where an ID was found, I suggest to avoid language like 'tool x detected IDs in the following locations:...', as we only know that a given tool detected an ID in at least one of these locations. Some tools only search specific sections of papers. So you could change language to something like 'Tool X detected X IDs. These IDs were present in the sections x,y,z' (or some other neutral language). You could also add a note of caution for readers to emphasise this point.

delwen commented on descriptive-analysis.qmd:
```diff
@@ -383,6 +518,18 @@ There were two cases of ctgov IDs reported with a missing digit, but the tools `
+-   **reg_id**: character. If reg_id_in_paper is TRUE or Unsure, this column registers the id or suspected id.
+
+-   **comments**: character. Brief observation on the flagged id.
+
```
@elisabascunan in the final part of the report (see the lines below), there are some super minor typos (e.g, there where, mention before).

delwen commented on descriptive-analysis.qmd:
```diff
@@ -175,41 +209,46 @@ The IDs analyzed could be in **one or more locations**, that is why there are mo
+data %>%
   group_by(PMCID) %>%
-  summarise(paper_is_research_article = any(paper_is_research_article))
-
-#Create a table with counts
-count_table <- result %>%
-  count(paper_is_research_article)
+  summarise(paper_is_research_article = any(paper_is_research_article)) %>%
+  count(paper_is_research_article) %>%
+  arrange(desc(paper_is_research_article)) %>%
+  kable()
```
this works, but an alternative could also be to first remove the PMCID duplicates and then count. As you prefer!

```
data %>%
    distinct(PMCID, .keep_all = TRUE) %>%
    count(paper_is_research_article) %>%
    arrange(desc(paper_is_research_article)) %>%
    kable()
```

delwen commented on descriptive-analysis.qmd:
```diff
@@ -34,6 +66,8 @@ library(ggplot2)
 library(wordcloud)
 library(RColorBrewer)
+library(knitr)
+library(tidyverse)
```
@elisabascunan would check and remove any packages that you no longer need, or are already part of another (e.g., readr in tidyverse).

delwen reviewed (changes requested):
Hi @elisabascunan, this is great! I added a couple of small comments for you to review. I also have an overarching comment about data structure: I suggest to create a subfolder `processed` under `data` where we store the completed extraction sheets. If you do this, you'll also have to adapt the paths in the analysis document (and anywhere else?). 

I will take care of the `data/raw` folder.

Commit:
data/processed folder created

datasets regset.csv and nc_man_check.csv were moved to this new folder

elisabascunan committed:
Updated descriptive-analysis.qmd file
- Modified location description
- Corrected minor typos
- Updated libraries used
- Modified the path to data file

elisabascunan commented:
@delwen the changes are ready and committed to the screen it report branch. Ready for pull.

delwen commented:
thanks @elisabascunan! The changes look great.

One last comment: in section 5 (description of each tool) just for Sciscore. Since this is the only tool that searches a specific section of the paper, I would add a breakdown of the counts for the locations where an ID was found. So something like the code I wrote below to show the overlap across locations and to drive home the differences for the methods section.

```
sciscore <- data |> filter(sciscore_hit)
sciscore |> count(id_in_abstract, id_in_methods, id_in_other_location)

id_in_abstract id_in_methods id_in_other_location     n
FALSE          FALSE         TRUE                     2
FALSE          TRUE          FALSE                   28
FALSE          TRUE          TRUE                     3
TRUE           TRUE          FALSE                   21
TRUE           TRUE          TRUE                     3
```

elisabascunan committed:
New table on Sciscore analysis
Added a table breaking down the location of IDs in methods section and/or another section.

elisabascunan commented:
@delwen very good point! I already made the commit with the table and a short description. Thank you for the observation.

delwen reviewed (approved):
looks great!
